src/app/malaria-101/activity-1/activity-1-1.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="row">
+	<div class="col-md-4 col-md-offset-3">
+		<video id="video" width="640" height="480" src="../../assets/videos/malaria.mp4" controls autoplay></video>
+	</div>
+</div>
+<div class="row">
+    <button prev class="col-lg-5 col-md-offset-2">
```

Comment:
I am not sure I understood the issue @yukie1412 . According to the angular directives tutorials 
https://angular.io/guide/attribute-directives
This is how it is being used.